wagtail/admin/forms/account.py

```diff
@@ -115,7 +122,55 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._original_avatar = self.instance.avatar
 
+    @contextmanager
+    def create_avatar_context(self, avatar):
+        avatar_context = willow.Image.open(avatar)
```

Comment:
Issue: I'm not sure this image is ever closed. Since this method is only used once, i'd suggest removing it and calling `Image.open` inside `reduce_avatar`.

Reply:
The willow image class does not have a close method, plus i only reused the pattern used in the wagtail/images/models to open the file. If i am to only open it in a variable, i have no certainty that the image would be open while all the functions are being performed on it. Please let me know if there is a different reason am not considering to remove the contextmanager from the code